tests/lib/configs/_rules.js

```diff
@@ -5,14 +5,11 @@
 "use strict"
 
 const { Linter } = require("eslint")
-const {
-    ConfigArrayFactory,
-} = require("@eslint/eslintrc/lib/config-array-factory")
-const Validator = require("eslint/lib/shared/config-validator")
-const { rules: removedRules } = require("eslint/conf/replacements.json")
+const { ConfigArrayFactory, ConfigValidator } = require("@eslint/eslintrc")
```

Comment:
I think we should import like this:

```suggestion
const { Legacy: { ConfigArrayFactory, ConfigValidator } } = require("@eslint/eslintrc")
```

Reply:
Why do you think that's better?
The root exported classes are exactly the same as the one's in the `Legacy` namespace

Reply:
I think only `Legacy` and `FlatCompat` are exported.

https://github.com/eslint/eslintrc/blob/438e8d7c522f4d5e04cb34f02e0ce232932a70e0/lib/index.js#L50-L56

Reply:
Hmmm really weird, my IDE said that they were exported by the root as well 🤔😅